ui/apps/platform/src/Containers/Violations/Details/Deployment/DeploymentTabWithoutReadAccessForDeployment.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactElement } from 'react';
+import { Card, CardBody, CardTitle, Flex, FlexItem } from '@patternfly/react-core';
+
+import { AlertDeployment } from 'types/alert.proto';
+
+import DeploymentOverview from './DeploymentOverview';
+
+export type DeploymentTabWithoutReadAccessForDeploymentProps = {
+    alertDeployment: AlertDeployment;
+};
+
+function DeploymentTabWithoutReadAccessForDeployment({
+    alertDeployment,
+}: DeploymentTabWithoutReadAccessForDeploymentProps): ReactElement {
```

Comment:
These 2 new wrapper component names are very verbose and coupled to the implementation details. Something more concise would be better.

```suggestion
function DeploymentTabWithoutPermission({
    alertDeployment,
}: DeploymentTabWithoutPermissionProps): ReactElement {
```